components/ui/TransactionButton.tsx

```diff
@@ -54,6 +54,7 @@ const TransactionButton: FC<TransactionButtonProps> = observer(
         onClick={(e) => click(e)}
         disabled={isDisabled()}
         data-test={dataTest}
+        type="submit"
```

Comment:
Have you verified that this change doesn't alter the behaviour of any of the other forms in the app?